drizzle-orm/src/neon-http/session.ts

```diff
@@ -1,76 +1,97 @@
 import type { FullQueryResults, QueryRows } from '@neondatabase/serverless';
+import type { BatchItem } from '~/batch';
```

Comment:
Here’s the other.

Reply:
I don’t know if there are reasons this isn’t feasible, but if you want to catch this with a tighter feedback loop, `--module nodenext` accomplishes that:

<img width="628" alt="image" src="https://github.com/drizzle-team/drizzle-orm/assets/3277153/587808ba-5ce9-4ba9-8b0b-eed6338ecae5">

N.B. `~/session` is also missing its extension, but doesn’t show up in attw because `PreparedQuery` doesn’t appear in the declaration emit for this file (it’s only used inside a function, not as a return/param type), so the import is elided.